app/jobs/download_and_extract_maxmind_file_job.rb

```diff
@@ -63,6 +63,6 @@ def untar(tarred_path)
         end
       end
     end
-    FileUtils.rm_rf tarred_path
+    # FileUtils.rm_rf tarred_path
```

Comment:
Should this still be commented out or just removed?

Reply:
We can probably remove, but I think leaving it commented for now is ok. For reference, it eliminates the race condition when spinning everything up on a single machine.